plutus-pab/src/Plutus/PAB/Core/ContractInstance.hs

```diff
@@ -90,7 +90,7 @@ activateContractSTM ::
 activateContractSTM runAppBackend a@ContractActivationArgs{caID, caWallet} = do
     activeContractInstanceId <- ContractInstanceId <$> uuidNextRandom
     logDebug @(ContractInstanceMsg t) $ InitialisingContract caID activeContractInstanceId
-    initialState <- Contract.initialState @t caID
+    initialState <- Contract.initialState @t activeContractInstanceId caID
```

Comment:
I suspect the `id` was added here to be able to log it?

Reply:
yes that's correct

Reply:
so when we write the logs to the terminal we can see which instance they came from (useful if there are multiple instances)